idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/proofs/intentions/implicitcoercion.kt

```diff
@@ -27,11 +27,12 @@ fun IdeMetaPlugin.makeImplicitCoercionIntention(compilerContext: CompilerContext
     }
   )
 
-fun KtElement.implicitParticipatingTypes(): Pair<KotlinType, KotlinType>? =
+fun KtElement.implicitParticipatingTypes(): List<PairTypes> =
   when (this) {
 
     is KtDotQualifiedExpression ->
```

Comment:
this is `safeAs` within a listOfNotNull

Reply:
Done, this simplifies the code a bit, thanks!